mobify/astro-tutorial
# Add eslint-import-resolver-webpack dev dependency

When running `npm run deps` on this branch, webpack fails and throws three resolver errors. These are fixed by adding `eslint-import-resolver-webpack`.

## Changes
- Added `eslint-import-resolver-webpack` as a dev dependency

## How to test-drive this PR
- Clone this branch
- Run `npm run deps`
- **Verify** the process completes successfully

## TODOS:
- [ ] Update tutorial documentation in the `dev` folder (in the Astro repo) to match the modified tutorial.
